test/integration/admin_login_test.rb

```diff
@@ -14,7 +14,7 @@ class AdminUserTest < ActionDispatch::IntegrationTest
     assert page.has_content?("Platform Admin")
```

Comment:
Maybe want to change this to assert_equal admin_dashboard_path, current_path? Platform admin should probably be visible from all admin pages, so this test doesn't ensure that upon login the admin lands on the right page.